src/Controller/UI/MdAppController.php

```diff
@@ -151,39 +172,42 @@ protected function renderUi()
     /**
      * Get logfile entries for particular dataset UDI.
      *
-     * @param string $udi The dataset UDI identifier.
+     * @param string      $udi         The dataset UDI identifier.
+     * @param MdappLogger $mdappLogger The Logger utility.
      *
-     * @Route("/getlog/{udi}")
+     * @Route("/mdapp/getlog/{udi}",
+     *      name="app_ui_mdapp_getlog"
```

Comment:
Non-standard route name. Use 'pelagos_app' prefix.  Also make sure this matched the old route.

Reply:
changed back